data/data/aws/vpc/vpc-public.tf

```diff
@@ -21,26 +25,29 @@ resource "aws_route_table" "default" {
 }
 
 resource "aws_main_route_table_association" "main_vpc_routes" {
+  count = var.vpc ? 0 : 1
+
   vpc_id         = data.aws_vpc.cluster_vpc.id
-  route_table_id = aws_route_table.default.id
+  route_table_id = aws_route_table.default.*.id
```

Comment:
https://www.terraform.io/upgrade-guides/0-12.html#working-with-count-on-resources

is route_table_id a list?

Reply:
> https://www.terraform.io/upgrade-guides/0-12.html#working-with-count-on-resources 

Ah, I see.  Fixed with 408576e5e -> d4d1cc3ac.